lib/blacklight/maps/maps_search_builder.rb

```diff
@@ -0,0 +1,20 @@
+module BlacklightMaps
+  module MapsSearchBuilder
```

Comment:
Does it make sense to make this a class rather than module? And subclass it from `Blacklight::Solr::SearchBuilder`?

That may would allow us to remove this logic: https://github.com/projectblacklight/blacklight-maps/pull/74/files#diff-544618e3e5d4681c08c1bf5fc29e5584R10